geonode/base/models.py

```diff
@@ -80,6 +80,24 @@
 
 logger = logging.getLogger(__name__)
 
+READ_PERMISSIONS = [
```

Comment:
Those arrays are duplicated. They have been already defined in utils.py.
Please do not duplicate code and variables.

Reply:
right, forgot about this artifact